l10n_ch_cash_rounding/models/account_move.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2023 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo import api, fields, models
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    invoice_cash_rounding_id = fields.Many2one(
+        "account.cash.rounding",
+        readonly=False,
+        compute="_compute_invoice_cash_rounding",
+        states={},
```

Comment:
```suggestion
        compute="_compute_invoice_cash_rounding",
        states={'draft': [('readonly', False)]},,
```
This will allow to define the cash rounding only if the move is not posted yet.